systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -722,6 +783,15 @@ protected void testDockerImagesForKafkaCluster(String clusterName, String namesp
     protected void afterEachMayOverride(ExtensionContext testContext) throws Exception {
         if (!Environment.SKIP_TEARDOWN) {
             ResourceManager.getInstance().deleteResources(testContext);
+
+            // if 'parallel namespace test' we are gonna delete namespace
+            if (testContext.getTags().contains(PARALLEL_NAMESPACE_TEST)) {
+
+                final String namespaceToDelete = mapWithNamespaces.get(testContext.getDisplayName());
+
+                LOGGER.info("Deleting namespace:{} for test case:{}", namespaceToDelete, testContext.getDisplayName());
+                cluster.deleteNamespace(namespaceToDelete);
+            }
```

Comment:
so if user won't specify this new tag, the tests won't be executed in parallel? I think it uses the tag in the wrong way. Wouldn't be better to check annotation itself?